src/docs/product/cli/dif.mdx

```diff
@@ -272,7 +272,7 @@ to embed it in a `sentry-debug-meta.properties` file. If you supply
 
 ```bash
 sentry-cli upload-proguard \
-    --write-properties app/build/intermediates/assets/release/sentry-debug-meta.properties \
+    --write-properties app/build/generated/assets/sentry/release/sentry-debug-meta.properties \
```

Comment:
```suggestion
    --write-properties app/build/generated/assets/sentry/{BuildVariant}/sentry-debug-meta.properties \
```

Reply:
@chizoba can you also update the bash above?

from:
```
sentry-cli upload-proguard \
    app/build/outputs/mapping/release/mapping.txt
```

to:
```
sentry-cli upload-proguard \
    app/build/outputs/mapping/{BuildVariant}/mapping.txt
```

since the `release` depends on the variant you have built